manager/integration/tests/test_ha.py

```diff
@@ -2641,9 +2692,42 @@ def prepare_engine_not_fully_deployed_evnironment_with_volumes():
     5. Wait for the engine image CR state become deploying
     """
 
+    volume1 = create_and_check_volume(client, "vol-1", size=str(3 * Gi))
+    volume2 = create_and_check_volume(client, "vol-2", size=str(3 * Gi))
 
-@pytest.mark.skip(reason="TODO") # NOQA
-def test_engine_image_miss_scheduled_perform_volume_operations():
+    lht_hostId = get_self_host_id()
+    for node in client.list_node():
+        if node.id != lht_hostId:
+            taint_node_id = node.id
+            break
+
+    core_api.patch_node(taint_node_id, {
+        "spec": {
+            "taints":
+                [{"effect": "NoSchedule",
+                  "key": "key",
+                  "value": "value"}]
+        }
+    })
```

Comment:
Looks duplicated to the node tainting in `prepare_engine_not_fully_deployed_evnironment`. Suggest abstracting to reusable function.